cclient/response/get_neighbors.h

```diff
@@ -27,11 +29,22 @@ typedef struct {
   int numberOfNewTransactions;
 } neighbor_t;
```

Comment:
We already have this structure in `gossip/neighbor.h`

Reply:
What is your suggestion? rename or use the neighbor_t in `common/network/neighbor.h`?
Why is it different from the [API Doc](https://iota.readme.io/v1.5.0/reference#getneighborsactivity)?

Reply:
I think these two structs are different in meaning, cclient's suppose to contain only URI, wherease gossip's contain more data about the participation of the neighbor in the protocol?
so if that is the case, it is ok to have both structs, with one being renamed

Reply:
I think only URI is fine for cclient too

Reply:
what is the plan for `gossip`? If the cclient is not depended on it. I think it would be ok with the same structure name and have different implementations. or what is the name you prefer?

Reply:
there is a slim chance  that both namespace will ever interleave, but i would rename the struct,
`neighbor_info_t` ?
`neighbor_id_t` ?